roadmap.md

```diff
@@ -0,0 +1,61 @@
+# WebDriver BiDi roadmap
+
+## Real-world end-to-end user scenarios
+
+This document presents an overview of real-world end-to-end user scenarios we aim to enable via the WebDriver BiDi protocol. Each scenario requires one or more WebDriver BiDi commands and events to be specified, tested, and implemented across browser engines.
+
+### A. Extracting content
+
+This scenario loads a web page and uses script evaluation to extract content on the page (e.g. the headlines). In spec terms, this involves:
+
+- [x] [handling sessions](https://w3c.github.io/webdriver-bidi/#module-session)
+- [x] [creating a new `browsingContext`](https://w3c.github.io/webdriver-bidi/#command-browsingContext-create)
+- [x] [navigating to a URL](https://w3c.github.io/webdriver-bidi/#command-browsingContext-navigate)
+- [x] [evaluating JavaScript in the page context](https://w3c.github.io/webdriver-bidi/#command-script-evaluate)
+- [x] [serialization/deserialization of JavaScript values](https://w3c.github.io/webdriver-bidi/#data-types-protocolValue)
+- [ ] [sandboxed script execution](https://github.com/w3c/webdriver-bidi/issues/144)
+
+### B. Logging
+
+This scenario loads a web page and uses BiDi event subscription to efficiently get notified about each `console.log` entry. **This is not possible with classic WebDriver.** In spec terms, this involves:
+
+- [x] (some items from the previous milestone)
+- [x] [the `log` module](https://w3c.github.io/webdriver-bidi/#module-log)
+
+### C. Submitting forms
+
+This scenario loads a web page, enter text into a form field via the keyboard, and submits the form via a mouse click before extracting the results from the page. In spec terms, this involves:
```

Comment:
What happens if the form also allows to be submitted by pressing the `Enter` key? Or in case of a touch screen, a tap. Would this scenario consider that as well?

Reply:
“emulating keyboard input” would support the Enter key as well. A tap is not explicitly part of this particular scenario but it would be supported through the same Input module.

The scenarios in this doc are just examples, hand-picked in an attempt for them to be easy-to-understand and somewhat common/realistic examples. They’re not meant to be exhaustive demonstrations of what’s possible.

Reply:
Thank you for clarifying. I also just read another comment where the actions API would be present in BiDi, so ideally that covers it.